server/index.js

```diff
@@ -57,6 +57,107 @@ app.get('/api/products/:productId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/cart', (req, res, next) => {
+  if (!req.session.cartId) {
+    res.json([]);
```

Comment:
Be sure to `return` here so your route doesn't continue on and try to query the database.

Reply:
fixed!